core/src/main/java/com/backblaze/b2/client/structures/B2AuthorizeAccountRequest.java

```diff
@@ -10,25 +10,25 @@
 
 public class B2AuthorizeAccountRequest {
     @B2Json.required
-    private final String accountId;
+    private final String applicationKeyId;
     @B2Json.required
     private final String applicationKey;
 
-    @B2Json.constructor(params = "accountId,applicationKey")
-    public B2AuthorizeAccountRequest(String accountId,
+    @B2Json.constructor(params = "applicationKeyId,applicationKey")
+    public B2AuthorizeAccountRequest(String applicationKeyId,
                                      String applicationKey) {
-        this.accountId = accountId;
+        this.applicationKeyId = applicationKeyId;
         this.applicationKey = applicationKey;
     }
 
-    public static Builder builder(String accountId,
+    public static Builder builder(String applicationKeyId,
                                   String applicationKey) {
-        return new Builder(accountId, applicationKey);
+        return new Builder(applicationKeyId, applicationKey);
 
     }
 
-    public String getAccountId() {
-        return accountId;
+    public String getApplicationKeyId() {
```

Comment:
looks like we're losing getAccountId() here too.
this whole trying-to-be-compatible is awkward.

Reply:
yea I can add it back.